js/private/test/BUILD.bazel

```diff
@@ -28,12 +28,15 @@ js_binary(
     entry_point = "shellcheck.js",
 )
 
-# For consistency on different platform, replace the host nodejs platform to linux_amd64
+# Make sed replacements for consistency on different platform
 genrule(
     name = "shell_launcher_sed",
     srcs = [":shellcheck_launcher"],
     outs = ["shellcheck_launcher_sed.sh"],
-    cmd = "cat $(execpath :shellcheck_launcher) | sed \"s#%s#linux_amd64#\" > $@" % host.platform,
+    cmd = "cat $(execpath :shellcheck_launcher) | sed \"s#$(BINDIR)#bazel-out/k8-fastbuild/bin#\" | sed \"s#%s#linux_amd64#\" | sed \"s#\\\"%s\\\"#\\\"k8\\\"#\" > $@" % (
```

Comment:
"k8"??

Reply:
You mean `linux_amd64`? That one is correct as it comes from our canonical platform naming in aspect_bazel_lib and not from bazel.

Reply:
no, there's also a k8 near the end of the regex, seems fishy to be linux/processor specific